modules/nf-core/openms/decoydatabase/meta.yml

```diff
@@ -0,0 +1,44 @@
+name: "openms_decoydatabase"
+description: Create a decoy peptide database from a standard FASTA database.
+keywords:
```

Comment:
Do you wanna include immunopeptidomics also as a keyword?

Reply:
The decoy database generation is not directly immunopeptidomics related I would say 🤔 I orientated myself on the structure of the test data repo for nf-core modules

![Screenshot 2024-02-26 at 16 38 38](https://github.com/nf-core/mhcquant/assets/43858870/cceb5501-19b2-4f51-8548-d2438918bd5c)